packages/create-app/templates/default-app/lerna.json.hbs

```diff
@@ -2,5 +2,5 @@
   "packages": ["packages/*", "plugins/*"],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "0.1.0"
+  "version": "{{version}}"
```

Comment:
Converted this file to a template and changed the version to be populated by the CLI.

One of the reports in discord was that create-plugin didn't work and it picked up the version`0.1.0` for @backstage/cli and @backstge/theme from here. The create-plugin doesn't work as these versions don't exist and it be pointing to the latest version.